trie/committer.go

```diff
@@ -250,6 +257,8 @@ func estimateSize(n node) int {
 		return 1 + len(n)
 	case hashNode:
 		return 1 + len(n)
+	case *rootNode:
+		return 1 + len(n.cachedHash) + len(n.TrieRoot) + len(n.ShadowTreeRoot)
```

Comment:
It may a estimateSize calculation, May `2 + 3 + 1+len(n.TrieRoot) + 1+len(n.ShadowTreeRoot)` = `7+len(n.TrieRoot) + len(n.ShadowTreeRoot)`?  `n.epoch`/2 bytes not `n.cachedHash`, may test it in rlp encode?